src/app.js

```diff
@@ -1,20 +1,24 @@
 import express from 'express';
-import { config } from 'dotenv';
 import cors from 'cors';
 import dbase from './database/config/database';
-import routes from './routes/index';
-import looger from './middlewares/logger';
-
-config();
```

Comment:
why not let dotenv config from here, which is the entry point since you removed it from the dev script

Reply:
The entry point was in src/database folder which is where different environment databases are setup and the connection is tested before running the server